test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -63,19 +63,24 @@ void
 call_log_manager::process_call_log_entry(json call_log_entry)
 {
     const std::string method_name = call_log_entry["method_name"].get<std::string>();
-    std::cout << "Processing entry " << method_name << std::endl;
     switch (_api_map.at(method_name)) {
     case api_method::wiredtiger_open:
         _conn = &connection_simulator::get_connection();
         break;
     case api_method::open_session:
+        if (_conn == nullptr){
```

Comment:
Can we assign _conn to `nullptr` in the constructor?

Reply:
I've assigned it in the initialiser.

Reply:
I think all the variables of a class should be assigned in a single place to avoid confusion.